guard/src/rules/functions/strings.rs

```diff
@@ -1,194 +0,0 @@
-use crate::rules::path_value::{Path, PathAwareValue};
```

Comment:
Why are these removed?

Reply:
Compiler was complaining about unused code, and i commented it out and ran the test suite + did manual testing to make sure it wasnt actually being used...Forgot to delete it. Good catch